resultsdbupdater/utils.py

```diff
@@ -13,21 +13,17 @@
 MAX_RESULT_DATA_SIZE = 8192
 
 
-def validate_data(data):
+def crop_data(data):
     """
-    Verifies whether data can be stored in ResultsDB.
-
-    Raises InvalidMessageError if data cannot be stored.
+    Crops large data values so they can be stored in ResultsDB.
 
     Even thought size for result data are not limited in database schema,
     Postgresql index has a size limited ("Values larger than 1/3 of a buffer
     page cannot be indexed").
     """
     for k, v in data.items():
         if isinstance(v, str) and len(v) > MAX_RESULT_DATA_SIZE:
-            raise exceptions.InvalidMessageError(
-                'Value for key "{0}" is too large (maximum size is {1})'.format(
-                    k, MAX_RESULT_DATA_SIZE))
+            data[k] = v[:MAX_RESULT_DATA_SIZE]
```

Comment:
Can we log that this is happening? Ideally, also include the message ID.

Reply:
+1 for logging this

Reply:
I've added logging and also ellipsis to the end of cropped text.

Reply:
BTW, the `msg.log` is set up to include the message ID.